app/services/bgs/people/request.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+
+module BGS
+  module People
+    class Request < Common::RedisStore
+      include Common::CacheAside
+
+      REDIS_CONFIG_KEY = :bgs_find_person_by_participant_id_response
+      redis_config_key REDIS_CONFIG_KEY
+
+      attr_reader :response
+
+      def find_person_by_participant_id(user:)
+        @response ||= find_person_by_participant_id_cached_response(user)
```

Comment:
is it useful to cache this response variable? if you pass in a different user then you will get the old user's cached data.

Reply:
Yeah that's a good point, I could see how this could potentially get called twice with different `user.participant_id` values. I'll change this